bitcoind-rpc-test/src/test/scala/org/bitcoins/rpc/v19/BitcoindV19RpcClientTest.scala

```diff
@@ -0,0 +1,62 @@
+package org.bitcoins.rpc.v19
+import org.bitcoins.rpc.client.common.BitcoindVersion
+import org.bitcoins.rpc.client.v19.BitcoindV19RpcClient
+import org.bitcoins.testkit.rpc.BitcoindRpcTestUtil
+import org.bitcoins.testkit.util.BitcoindRpcTest
+
+import scala.concurrent.Future
+
+class BitcoindV19RpcClientTest extends BitcoindRpcTest {
```

Comment:
I think we are missing an `afterAll` method here shutting things down

Reply:
None of the other `RpcClientTest`s have one, should I add one to them in a future PR or do I not need one?

Reply:
@Christewart how is it that we don't have `afterAll`s in these tests? Am I missing something or is this definitely a problem?

Reply:
Take a look at the `BitcoindRpcTest` trait